download_controller.py

```
@@ -0,0 +1,86 @@
+
+class DownloadControllerAsync:
+    def __init__(self, view, service):
+        self.view = view
+        self.download_service = service
+    
+    async def handle_rss_comic_clicked(self, comic_dict):
+        self.comic_dict = comic_dict
+        self.view.update_status(f"Starting download of: {comic_dict['title']}")
+        try:
+            download_link = self.download_service.get_download_links(comic_dict.get("link"))
+            filepath = self.download_service.download_comic(download_link)
```

Flake8 (7.2.0)

[error] local variable 'filepath' is assigned to but never used

(F841)